contrib/remote-dev/conf/job.template

```diff
@@ -0,0 +1,32 @@
+protocolVersion: 2
+name: remote_dev_job_template
+type: job
+jobRetryCount: 0
+prerequisites:
+  - type: dockerimage
+    uri: openpai/pai.example.tensorflow
+    name: docker_image_0
+taskRoles:
+  Task_role_1:
+    instances: 1
+    completion:
+      minFailedInstances: 1
+      minSucceededInstances: 1
+    dockerImage: docker_image_0
+    resourcePerInstance:
+      gpu: 0
+      cpu: 1
+      memoryMB: 2048
+    commands:
+      - '`#REMOTE_DEV_START`'
```

Comment:
```suggestion
      # REMOTE_DEV_START
```

Reply:
Can this work now? I remember that this cannot work before.

Reply:
this is comment in yaml, not shell command
the comment is read by user, not shell